src/app/common/config/translations.ts

```diff
@@ -48,14 +48,14 @@ const translations = {
           texts: ['Removing your validator will cause your operators to stop managing it in your behalf, which will result in its inactivation (penalties on the Beacon Chain).', 'Please note that this action only applies to its removal from our network and does not exit your validator from the Beacon Chain.'],
           warningMessage: 'To avoid slashing, it is advised to wait at least 2 epochs prior to running the validator on an alternative service.',
           checkBoxes: ['I understand that my validator will be removed from the network and it will stop attesting on the beacon chain'],
-          buttonText: (count: number) => `Remove ${count} validators`,
+          buttonText: (count: number, isLoading: boolean) => `${ isLoading ? 'Removing' : 'Remove'} ${count > 1 ? `${count} validators` : 'validator' }`,
```

Comment:
you can dynamically create this string by creating a function that gets a string template and variadic params and just returns the entire text. then you can cal it here and in line 58